cupy/linalg/norms.py

```diff
@@ -240,17 +240,20 @@ def _slogdet_one(a):
     buffersize = getrf_bufferSize(handle, m, m, a_copy.data.ptr, m)
     workspace = cupy.empty(buffersize, dtype=dtype)
     getrf(handle, m, m, a_copy.data.ptr, m, workspace.data.ptr,
-          ipiv.data.ptr, info.data.ptr)
+          ipiv.data.ptr, dev_info.data.ptr)
+
+    try:
+        cupy.linalg.util._check_cusolver_dev_info_if_synchronization_allowed(
+            getrf, dev_info)
```

Comment:
Open for discussion. C.f. https://github.com/cupy/cupy/pull/2437/files#diff-2ef2aeb8e00bf17e47e54407373d8c27R163-R164.